RELEASE_PROCESS.md

```diff
@@ -20,92 +40,278 @@ Regular releases concern only major and minor releases. We do not decide up fron
+
+_how is this done?_
+
+Example release candidate
+https://github.com/asyncapi/spec-json-schemas/pull/151
+
+
+### Step 14 - merge the release branches
+
+Once everything is ready, it is time to merge the [release branches](#step-3---create-release-branches) using the [draft pull requests prepared earlier](#step-7---create-draft-pull-requests).
+
+Merging can only be done by [code owners](#code-owners).
+
+Pull requests must have a `feat: ` prefix that, after the merge, produces a release candidate with a minor version update. The major version must have `feat!: ` prefix.
+
+Pull requests in the `parser-js` can be merged only if it uses the release candidate of `@asyncapi/specs` package produced after merge of a pull request in `spec-json-schemas`.
+
+First, changes are merged into the `spec` repository, then `spec-json-schemas` and then in `parser-js`.
+
+Release means merge of pull requests created from a release branch against the master branch. First, changes are merged into the `spec` repository, then `spec-json-schemas` and at the end in `parser-js`. Like in the case of the merge of release candidates, a pull request in `parser-js` can be merged only if it uses the final release of the new `@asyncapi/specs` package.
+
+
+### Step 15 - publish releases
+
+The [release coordinator](#%22release-coordinator%22) should ask the [code owners](#code-owners) for each repository to create the release in Github, using the [release notes they have prepared](#step-10---prepare-release-notes).
+
+
+### Step 16 - notify tool maintainers
+
+The [release coordinator](#%22release-coordinator%22) should notify maintainers of the following repositories that the first feature is merged and that release will be produced and therefore they need to start preparing for it:
   - [JavaScript Converter](https://github.com/asyncapi/converter-js/)
   - [Playground](https://github.com/asyncapi/playground/)
   - [React component](https://github.com/asyncapi/asyncapi-react/)
   - [Markdown template](https://github.com/asyncapi/markdown-template)
 
-### Steps only after first feature merge
+The [release coordinator](#%22release-coordinator%22) should also make sure other maintainers from other projects under the AsyncAPI GitHub organization released their packages.
+
+
+### Step 17 - notify the community
 
-- Open draft pull requests in all repositories required by the contribution guide. They should point from the release branch in the upstream to the master branch in the upstream,
-- Update documentation in [website](https://github.com/asyncapi/website) to promote latest version of the specification. You can do it in the same pull request with release notes. No need to care for the specification markdown file on the Website. It updates automatically during work on release candidates and also after the final release.
+Every release of the release candidate is automatically published on the AsyncAPI Twitter account and in the releases-dedicated Slack channel.
```

Comment:
please advice that release coordinator should be proud of the work done, and announce release from private accounts on Twitter and LinkedIn and then these will be used as a base for official announcement to assure release coordinator participation